packages/twenty-server/src/engine/metadata-modules/field-metadata/validators/is-valid-graphql-enum-name.validator.ts

```diff
@@ -4,7 +4,7 @@ import {
   ValidationArguments,
 } from 'class-validator';
 
-const graphQLEnumNameRegex = /^[_A-Za-z][_0-9A-Za-z]+$/;
+const graphQLEnumNameRegex = /^[_A-Za-z][_0-9A-Za-z]*$/;
```

Comment:
@magrinj are you ok with this update? One-char long names are not an issue with graphql